collections/soldiers.js

```diff
@@ -0,0 +1,36 @@
+const MongoClient = require('mongodb').MongoClient;
```

Comment:
not only that we trained you to work in tdd and bdd, but we emphasized it's importance in this assignment. the tests should have been the first commit, not the very last.